package.json

```diff
@@ -21,12 +24,12 @@
 		"typescript-json-schema": "^0.48.0"
 	},
 	"dependencies": {
-		"glob": "^7.1.7",
 		"js-yaml": "^4.1.0",
 		"knockout": "^3.5.1",
 		"meriyah": "^3.1.6",
 		"source-map": "^0.7.3",
 		"strip-json-comments": "^3.1.1",
+		"tiny-glob": "^0.2.8",
```

Comment:
Tiny-glob is faster and smaller. The package does create a deprecation warning when run in Node 16. Any thoughts?

```
(node:21656) [DEP0128] DeprecationWarning: Invalid 'main' field in 'C:\...\kolint\node_modules\tiny-glob\package.json' of 'src/index.js'. Please either fix that or report it to the module author
```